2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -436,38 +393,21 @@ print(pData(pep_res))
 #> data frame with 0 columns and 4 rows
 ```
 
+
 ```{r}
 # Add replicate number to phenotype data
-pData(pep_res)$Replicate <- 1:ncol(exprs_data)
+pData(pep_res)$Replicate <- as.numeric(colnames(exprs_data))
 ```
 
 
-```{r}
-print(pep_res)
-#> MSnSet (storageMode: lockedEnvironment)
-#> assayData: 3839 features, 4 samples 
-#>   element names: exprs 
-#> protocolData: none
-#> phenoData
-#>   sampleNames: ratio_1 ratio_2 ratio_4 ratio_3
-#>   varLabels: Replicate
-#>   varMetadata: labelDescription
-#> featureData
-#>   featureNames: AAAAAAAAAAAAAAAGAGAGAK_ AAAAAAAAAAGAAGGR_1xAcetyl
-#>     [N-Term] ... YYVTIIDAPGHR_ (3839 total)
-#>   fvarLabels: Master.Protein.Accessions Sequence ... Matched_NC_3 (7
-#>     total)
-#>   fvarMetadata: labelDescription
-#> experimentData: use 'experimentData(object)'
-#> Annotation:  
-#> - - - Processing information - - -
-#>  MSnbase version: 2.20.4
-```
 
 ```{r}
+pep_res_flt <- pep_res %>%
+  camprotR::restrict_features_per_protein(min_features=2, plot=FALSE)
```

Comment:
Desirable to have a step to ensure that peptides are filtered to remove 'one-hit wonders', e.g protein-level quantification coming from just a single peptide.